.azure_pipelines/ci-pipeline-makefile.yml

```diff
@@ -81,6 +81,15 @@ jobs:
           MYST_SKIP_PR_TEST: 1
           MYST_DO_CLEANUP: 1
 
+      # run PyTorch tests
+      - script: |
+          make tests -C $(Build.SourcesDirectory)/solutions/pytorch_tests
```

Comment:
The pipeline didn't execute this stage. It complains: `make: *** No rule to make target 'tests'`.

Should it be `make run ...`?

Reply:
Good catch, I only tested it in Jenkins. Forgot to fix that here.

Reply:
Fixed.